rplugin/python3/deoplete/sources/go.py

```diff
@@ -34,8 +34,12 @@ def __init__(self, vim):
         self.sort_class = self.vim.vars['deoplete#sources#go#sort_class']
 
     def get_complete_position(self, context):
-        m = re.search(r'\w*$', context['input'])
-        return m.start() if m else -1
+        m = re.search(r'"[^"]*$', context['input'])
```

Comment:
Maybe, this regex is better than it. (slightly tested)
```python
m = re.search(r'(\w*$|(?<=")([\.\/\w]*$))', context['input'])
```
Not required `if...else` if use this regex.
Could you test it?